constraint/config/crds/templates.gatekeeper.sh_constrainttemplates.yaml

```diff
@@ -79,6 +79,9 @@ spec:
                             description: 'The engine used to evaluate the code. Example:
                               "Rego". Required.'
                             type: string
```

Comment:
`generateVAP` should be a property of the VAP engine, not the CT schema. It should be a member of `source` for the k8snative driver.

Otherwise, we couple CTs to VAP, and only the VAP engine is intended to generate VAP objects anyway

Reply:
@maxsmythe good point, moved `generateVAP` field to `source` for k8snative driver.